mona_sdk/client.py

```diff
@@ -114,17 +116,21 @@
     "SAMPLING_FACTORS_MAX_AGE_SECONDS", 300
 )
 
-SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service."
-UPLOAD_CONFIG_ERROR_MESSAGE = (
-    "Could not upload the new configuration, please check it is valid."
+UNAUTHENTICATED_ERROR_CHECK_MESSAGE = (
```

Comment:
The suffix should be ERROR_MESSAGE so please rename to UNAUTHENTICATED_CHECK_ERROR_MESSAGE